include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -130,6 +130,26 @@
 namespace onnxruntime {
 namespace concurrency {
 
+class Profiler {
+ public:
+  Profiler() = default;
+  ~Profiler() = default;
+  ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(Profiler);
+  using CLOCK = std::chrono::high_resolution_clock;
+  operator bool() const;
+  void Start();
```

Comment:
would be good to have some instructions about usage. i guess End() shouldn't be called before Start(), how about multiple calls to Start()?

Reply:
Add comments on the usage.

---
In reply to: [579575920](https://github.com/microsoft/onnxruntime/pull/6748#discussion_r579575920) [](ancestors = 579575920)